docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+### Package Signing
+
+When enabled, signs the _.APK_ file using the keystore details. This is disabled by default.
+
+## iOS
+
+These are iOS-specific .NET MAUI settings.
+
+### Build
+
+Settings related to building the iOS app.
+
+- **Linker behavior**
+
+  The linker can strip out unused methods, properties, fields, events, structs, and even classes in order to reduce the overall size of the application. You can add a `Preserve` attribute to any of these in order to prevent the linker from stripping it out if it's needed for serialization or reflection.
+
+  > [!WARNING]
+  > Enabling this feature may hinder debugging, as it may strip out property accessors that would allow you to inspect the state of your objects.
+
+  Options are:
+
+  - `Don't link`
+  - `Link Framework SDKs only` (default)
+  - `Link All`
+
+- **LLVM**
+
+  When enabled, uses the LLVM optimized compiler. This can be set for individually for both Debug and Release modes.
+
+  The default is enabled for Release builds.
+
+- **Float operations**
+
+  Performs all 32-bit float operations as 64-bit float operations.
+
+- **Symbols**
+
+  When enabled, strips native debugging symbols from the output. This is enabled by default.
+
+- **Garbage collector**
+
+  When enabled, uses the concurrent garbage collector. This is disabled by default.
+
+- **Additional arguments**
+
+  Additional command line arguments to be passed to the application bundling code.
+
+- **Optimization**
+
+  When enabled, optimizes _.PNG_ images. This is enabled by default.
+
+### Bundle Signing
+
+These settings are related to generating and signing the app bundle.
+
+- **Scheme**
+
+  Configures the signing scheme for the bundle. It can be set to one of the following values:
+
+  - `Manual provisioning`: With this value, you'll be responsible for setting provisioning profiles and signing certificates yourself.
+  - `Automatic provisioning`: (default) With this value, Visual Studio will set provisioning profiles and signing certificates for you to simplify app testing on a device.
+
+- **Signing identity**
+
+  A signing identity is the certificate and private key pair that's used for code-signing app bundle using Apple's codesign utility.
+
+  - `Developer (automatic)` (default)
+  - `Distribution (automatic)`
+
+- **Provisioning profile**
+
+  Provisioning profiles are a way of tying together a team of developers with an App ID and, potentially, a list of test devices. The provisioning profiles list is filtered to only show provisioning profiles that match both the chosen identity and the App ID (aka bundle identifier) set in the _Info.plist_. If the provisioning profile that you're looking for isn't in the list, make sure that you've chosen a compatible identity and double-check that the bundle identifier set in your _Info.plist_ is correct.
+
+- **Custom Entitlements**
```

Comment:
Link to https://docs.microsoft.com/en-us/dotnet/maui/ios/deployment/entitlements?

It's not that great at the moment, but it will get expanded at some point.